content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,281 @@
+Configuration Database. Further, they should ensure that administrators are
+subscribed to the relevant mailing lists.
+
+#### Resource Centre Security Officer
+
+The person responsible for keeping the site compliant with the
+[EGI security policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies). They are
+the primary contact for the NGI Security officer and
+[EGI Computer Security Incident Response Team (CSIRT)](https://csirt.egi.eu).
+The Site Security Officer deals with security incidents and shall respond to
+enquiries in a timely fashion as defined in the collection of
+[security procedures and policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies).
+
+### Regional level
+
+#### Regional Operator on Duty (ROD)
+
+A team responsible for solving problems and incidents in the infrastructure
+according to agreed procedures. ROD teams monitor the Resource Centres in their
+region, react to incidents identified by the monitoring tools, and oversee
+incidents and related problems through to their resolution. They ensure that
+incidents are properly recorded and that the solutions progress according to
+specified time lines. They also provide support to Resource Centres and Virtual
+Organisations (VOs) and provide support to oversight bodies in cases of
+unresponsive Resource Centres. They ensure that all the necessary information is
+available to all parties. The team is provided by each NGI and requires
+procedural knowledge on the process (rather than technical skills) for their
+work. New ROD team members are required to read the
+[ROD Welcome page](https://wiki.egi.eu/wiki/Regional_Operator_on_Duty_welcome)
+and be familiar with
+[ROD wiki page](https://wiki.egi.eu/wiki/Regional_Operator_on_Duty).
+
+#### NGI Security officer
+
+An NGI Security Officer is responsible for overseeing the security related
+aspects of the operations of the NGI.
+
+NGIs and Sites **MUST** respond in a timely manner to the security requests and
+alerts coming from the [EGI CSIRT](#egi-csirt) and [EGI IRTF](#egi-irtf).
+
+#### NGI Operations manager
+
+The NGI Operations manager is the contact point for all operational matters and
+represents the NGI within the
+[Operations Management Board (OMB)](https://confluence.egi.eu/display/EGIBG/Operations+Management+Board).
+
+They are mainly responsible for:
+
+- keeping up to date the NGI entry in the Configuration Database and managing
+  the status of all sites under their NGI, and ensuring their information is
+  also kept current
+- addressing problems with Site availability or reliability. The reports are
+  issued on a monthly basis and the NGI operations managers have 10 days to
+  respond to identified problems
+- attending regular
+  [Operations Management Board meetings](https://indico.egi.eu/category/19/)
+
+All NGI operations management responsibilities are listed in the
+[Resource Infrastructure Provider OLA document](https://documents.egi.eu/document/463).
+
+### Project level
```

Comment:
@paolini78 , @mviljoen-egi I find this quite unclear, and probably outdated, it's no more about an EGEE project, but it's about the EGI infrastructure. May be we could replace by at the infrastructure level or at a global level.

Reply:
it is how the layers are named in GOCDB. Of course we can replace it.

Reply:
Yes I know, but I think it's better to use something that will make more sense like infrastructure or global. Maybe global is more meaningful/in opposition to the regional level?

Reply:
ok